lib/tasks/downloads.rake

```diff
@@ -41,19 +41,23 @@ task :downloads => [:environment, :windows_downloads, :mac_downloads]
 
 task :windows_downloads => :environment do
   # find latest windows version
-  project = "msysgit"
-  win_downloads = file_downloads_from_github("msysgit/msysgit")
+  project = "git-for-windows"
+  win_downloads = file_downloads_from_github("git-for-windows/git")
   win_downloads.each do |url, date|
     name = url.split('/').last
-    if m = /^Git-(.*?)-(.*?)(\d{4})(\d{2})(\d{2})\.exe/.match(name)
-      version = m[1]
+    if m = /^(.*?)Git-(.*?)-(.*?)-(.*?)-(.*?)-bit\.(.*?)exe/.match(name)
+      portable = m[1]
+      version = m[2]
```

Comment:
Wait, never mind.  Brain fart.

It might be a good idea to document the name system in a comment next to the regex, though.